backend/src/middleware/index.js

```diff
@@ -10,7 +10,7 @@ import user from './userMiddleware'
 import includedFields from './includedFieldsMiddleware'
 import orderBy from './orderByMiddleware'
 import validation from './validation/validationMiddleware'
-import notifications from './notifications'
+import handleContentData from './handleHtmlContent/handleContentData'
```

Comment:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-07-01T22:39:29Z" title="Tuesday, July 2nd 2019, 12:39:29 am +02:00">Jul 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -11,7 +11,7 @@ import user from './userMiddleware'
 import includedFields from './includedFieldsMiddleware'
 import orderBy from './orderByMiddleware'
 import validation from './validation'
-import notifications from './notifications'
+import handleContentData from './handleHtmlContent/handleContentData'
```


maybe `./handleHtmlContent/...` is the appropriate location for all the content analysis functions like ` extractMentionedUsers`

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-02T06:10:59Z" title="Tuesday, July 2nd 2019, 8:10:59 am +02:00">Jul 2, 2019</time>_

---


But that’s all already in `./handleHtmlContent/…` ! @ulfgebhardt